app/server_test.go

```diff
@@ -548,7 +548,7 @@ func TestPanicLog(t *testing.T) {
 	s, err := NewServer(SetLogger(logger))
 	require.NoError(t, err)
 
-	// Route for just panicing
+	// Route for just panicking
```

Comment:
This is referring to the "panic" feature in the language. Not to be confused with the english word "panic". So let's leave this one as-is, even if it's not a valid word.